core/dbt/include/global_project/macros/materializations/models/incremental/incremental.sql

```diff
@@ -52,7 +52,7 @@
     {% set incremental_strategy = config.get('incremental_strategy') or 'default' %}
     {% set incremental_predicates = config.get('incremental_predicates', none) %}
     {% set strategy_sql_macro_func = adapter.get_incremental_strategy_macro(context, incremental_strategy) %}
-    {% set strategy_arg_dict = ({'target_relation': target_relation, 'temp_relation': temp_relation, 'unique_key': unique_key, 'dest_columns': dest_columns, 'predicates': incremental_predicates }) %}
+    {% set strategy_arg_dict = ({'target_relation': target_relation, 'temp_relation': temp_relation, 'unique_key': unique_key, 'dest_columns': dest_columns, 'incremental_predicates': incremental_predicates }) %}
```

Comment:
@jtcohen6 / @VersusFacit - renaming `predicates -> incremental_predicates` this would be a breaking change right?

Reply:
Yes, it would be a breaking change in the signature of macros like `get_merge_sql`, and so it would require an update from anyone who's copy-paste-overridden those macros with custom versions in their own projects.

I believe there's a way to around this, in Jinja, by doing something like:
```
{% macro some_func(arg1, arg2_renamed=None) %}
  {% if kwargs.get('arg2_oldname') %}
    {% set arg2_renamed = arg2_oldname %}
  {% endif %}
{% endmacro %}
```

If we can't find a way to go like that, this is something we can document + call out explicitly in the upgrade/migration guide for the release including this change.

Reply:
excellent, will attempt this! would that require a second test to ensure either predicates or incremental_predicates pass?

Reply:
I think that would be good, yep!